packages/midscene/src/ai-model/prompt/element_inspector.ts

```diff
@@ -139,6 +143,36 @@ Output Example:
   `;
 }
 
+// claude 3.5 sonnet computer The ability to understand the content of the image is better, Does not provide element snapshot effect
+export function systemPromptToFindElementPosition() {
+  return `
+    ## Role:
+    You are an expert in software page image (2D) and page element text analysis.
+
+    ## Objective:
+    Based on screenshots and descriptions, find specific coordinates
+
+    ## Output Format:
+
+    Please return the result in JSON format as follows:
+
+    \`\`\`json
+    {
+      "elements": [
+        {
+          // Describe the reason for finding this element, replace with actual value in practice
+          "reason": "Reason for finding element 4: It is located in the upper right corner, is an image type, and according to the screenshot, it is a shopping cart icon button",
+          "text": "",
```

Comment:
should we still keep the 'text' field ? 

If we decide to keep it, could you explain how the AI would use this ‘text’ field?